### DISCUSSION TITLE
Keep drone upside down

### DISCUSSION THREAD

**Discussion by kwanyongcho7** [microsoft/AirSim#3739] [Support Q&A]
I am using Px4 with HITL, and I want to keep my drone upside down when it collided.

Now, Airsim automatically fixes upside down, which my drone thinks it is still upside down.. 

Is there any way I can remove automatic fix for upside-down?

**Comment by zimmy87** [comment]
This can be achieved by disabling ground lock in the FastPhysicsEngine setting:

```
{
  "SeeDocsAt": "https://github.com/Microsoft/AirSim/blob/master/docs/settings.md",
  "SettingsVersion": 1.2,
  "PhysicsEngineName": "FastPhysicsEngine",
  "FastPhysicsEngine": {
      "EnableGroundLock": false
  }
}
```
